example/ck_tile/17_grouped_gemm/grouped_gemm.hpp

```diff
@@ -34,7 +34,20 @@ using grouped_gemm_kargs = ck_tile::GroupedGemmHostArgs;
 auto create_args(int argc, char* argv[])
 {
     ck_tile::ArgParser arg_parser;
-    arg_parser.insert("a_layout", "R", "A tensor data layout - Row by default")
+    arg_parser
+        .insert("Ms", "", "The Ms data size, might be greater than 1 - it is empty by default.")
+        .insert("Ns", "", "The Ns data size, might be greater than 1 - it is empty by default.")
+        .insert("Ks", "", "The Ks data size, might be greater than 1 - it is empty by default.")
+        .insert("stride_As",
+                "",
+                "The stride As data size, might be greater than 1 - it is empty by default.")
+        .insert("stride_Bs",
+                "",
+                "The stride Bs data size, might be greater than 1 - it is empty by default.")
+        .insert("stride_Cs",
+                "",
+                "The stride Cs data size, might be greater than 1 - it is empty by default.")
```

Comment:
Those comments are misleading and confusing. Please make them clear and related to argument.